### PR TITLE
NEW FEATURES MADAFAKA

### PR DESCRIPTION
**Pull request by abyxalex**

### CODE REVIEW

**Comment by abyxalex** [comment]
just a comment

File: miembros.md [hunk 1]
```diff
@@ -22,4 +22,4 @@
 20. Lady Shiva
 21. Giganta
 22. Deadshot
-23. Scarecrow
```
Comment (alexabyx) [inline miembros.md:25]: revert this shit

**Review by alexabyx — CHANGES_REQUESTED** [review]
rever the shit

**Comment by abyxalex** [comment]
yeah sorry

File: miembros.md (hunk 1 above)
Comment (abyxalex) [reply miembros.md:25]: yeah sorry

**Review by alexabyx — APPROVED** [review]
nice job

**Pull request merged**